valhalla/proposals/models.py

```diff
@@ -103,6 +114,7 @@ class TimeAllocation(models.Model):
     semester = models.ForeignKey(Semester)
     proposal = models.ForeignKey(Proposal)
     telescope_class = models.CharField(max_length=20, choices=TELESCOPE_CLASSES)
+    instrument_name = models.CharField(max_length=200, choices=INSTRUMENT_NAMES)
```

Comment:
can we call this variable something other than instrument_name? instrument_name often means the specific instrument's name (i.e. fl06). What about camera_type, or instrument_type, or instrument_class or camera_class?

Reply:
It's named `instrument_name` in Molecule, so I'd prefer to keep things consistent, at least within Valhalla. I'm aware that it is inconsistent in other places in the stack.

Reply:
good point